corehq/apps/app_manager/management/commands/benchmark_direct_ccz.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+
+from django.core.management import BaseCommand
+
+from corehq.apps.app_manager.dbaccessors import get_app
+from corehq.apps.app_manager.management.commands.benchmark_build_times import Timer
+from corehq.apps.app_manager.views.cli import get_direct_ccz
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'domain_app_id_pairs',
+            help='A JSON list where each element has the format [<domain>, <app_id>]',
+            type=json.loads,
+        )
+
+    def handle(self, domain_app_id_pairs, **options):
+        for (domain, app_id) in domain_app_id_pairs:
+            print("%s: %s" % (domain, app_id))
+            with Timer():
```

Comment:
Is the reason you are doing these like this so that you have some data to show someone externally?
Otherwise, have you checked out: https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/decorators/profile.py ? 
I think just having simple timing data might not help you narrow down where the issues are coming from, but maybe you already have a way you are doing this. 

You could possibly also use `TimingContext` to give you more granular, but similar data. This is going to give you back the time to both get the app and test the `get_direct_ccz` method, so how will you know which one is taking time

Reply:
> Is the reason you are doing these like this so that you have some data to show someone externally?

Yeah, right now this is for benchmarking, not profiling

> Otherwise, have you checked out: https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/decorators/profile.py ?

I used this when I was first analyzing "Make New Version" - helped me identify duplicated code being executed.  I haven't profiled ```download_ccz``` yet. It's probably easy to add optional profiling to the two benchmarking commands, which could be useful for iteratively profiling.

> This is going to give you back the time to both get the app and test the get_direct_ccz method, so how will you know which one is taking time

In principle we care about both, since that's what ```direct_ccz``` does.  However for big apps the ```get_direct_ccz``` step takes the large majority of the time.

Reply:
@proteusvacuum Added the option to profile